packages/editor-ui/src/components/Node.vue

```diff
@@ -320,9 +321,23 @@ const nodeTitle = computed(() => {
 const waiting = computed(() => {
 	const workflowExecution = workflowsStore.getWorkflowExecution as ExecutionSummary;
 
-	if (workflowExecution?.waitTill) {
+	if (workflowExecution?.waitTill && !workflowExecution?.finished) {
```

Comment:
These changes should probably also apply to the new canvas
`packages/editor-ui/src/composables/useCanvasMapping.ts` L317

Reply:
this condition is actually not needed, I did remove it